src/Memory/MemoryUtils.cpp

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "MemoryUtils.h"
+
+#include <absl/strings/numbers.h>
+#include <absl/strings/str_split.h>
+#include <stdlib.h>
+
+#include <string>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/ReadFileToString.h"
+#include "OrbitBase/Result.h"
+#include "OrbitBase/ThreadUtils.h"
+
+using orbit_grpc_protos::TotalMemoryUsageInfo;
+
+std::optional<TotalMemoryUsageInfo> ParseMemInfo(const std::string& meminfo_content) {
```

Comment:
Nit: `ParseMemoryInfo` and `memory_info_content`.  Per style guide, the use of abbreviations should be minimal.
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
The reason for the name of the function is that the file is called `/proc/meminfo`.

Reply:
Makes sense.